tools/recetox_aplcms/mzml_id_getter.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+from pymzml.run import Reader
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Get run ID from an mzML file.')
+    parser.add_argument('mzml_file', help='Path to an mzML file to get run ID from.')
+    args = parser.parse_args()
+
+    mzml = Reader(args.mzml_file)
+    id = mzml.info['run_id']
+
+    if id is None:
+        sys.stderr.write('ERROR: The mzML file has no run ID.')
+        sys.exit(1)
```

Comment:
The `remove_noise` wrapper will fail when the `run` `ID` is not present. We discussed that it should just write a warning since this causes issues for further steps in the workflow, but the wrapper itself can work. Any ideas on how this should then behave?

Reply:
Covered in [273fe3f](https://github.com/RECETOX/galaxytools/pull/317/commits/273fe3fa52c256e5421d9ed263359018df6c50ee).